build_downstream.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# This script is used to build downstream srpm for adb-utils after minor update
+# to upstream.
+# Note: Make sure you update *version* detail before run this script.
+
+UPSTREAM_NAME=adb-utils
+VERSION=1.1
+DOWNSTREAM_NAME=cdk-utils
+SOURCE=https://github.com/projectatomic/$UPSTREAM_NAME/archive/v${VERSION}.tar.gz
+SPEC_SRC="https:\/\/github.com\/projectatomic\/\%{name}\/archive\/v\%{version}\.tar\.gz"
+
+# Change to downstream name and source
+sed -i -e "s/Name:\(.*\)adb-utils/Name:\1cdk-utils/" ${UPSTREAM_NAME}.spec \
+       -e "s/Source0:\(\s.*\)${SPEC_SRC}/Source0:\1\%{name}-\%{version}\.tar\.gz/"
+
+mv ${UPSTREAM_NAME}.spec ${DOWNSTREAM_NAME}.spec
+wget $SOURCE
```

Comment:
wget is fine, how about using `spectool` to pull in sources? since spectool is designed for downloading sources and patches from specfiles?